docs/en/tutorial/tutorial_sql_2.md

```diff
@@ -51,98 +52,111 @@ OpenMLDB currently supports`LAST JOIN`to perform database like join operations.
 Then, we can do the following join operation:
 
 ```sql
--- des c: In view of LAST JOIN splicing based on ORDER BY
 SELECT * FROM s1 LAST JOIN s2 ORDER BY s2.std_ts ON s1.col1 = s2.col1;
 ```
 
-As shown below, when `LAST JOIN` configure in `ORDER BY`，right click `std_ts` sort and splice the last hit data row. Take the second behavior of the left table as an example. There are 2 qualified right tables. Press `std_ts` after sorting and select the last item `3, b, 2020-05-20 10:11:13` .
+As shown below, left table `LAST JOIN` right table with `ORDER BY` and right table is sorted by  `std_ts`. Take the second row of the left table as an example. There are two matched rows in right table. After sorting by `std_ts` the last row `3, b, 2020-05-20 10:11:13` is selected.
 
 ![img](images/lastjoin_1.jpg)
 
 ![img](images/lastjoin_2.jpg)
 
-## 3. Multi-Row Aggregation Feature of Sub Table
+## 3. Multi-Row Aggregation  over Multiple Tables
 
-For the sub table splicing scenario, OpenMLDB extends the standard window syntax and adds [WINDOW UNION]( http://docs-cn.openmldb.ai/2620896) characteristic. It supports splicing multiple pieces of data from the sub table to form a sub table window. Based on the sub table splicing window, it is convenient to construct the multi-row aggregation feature of the sub table. Similarly, two steps need to be completed to construct the multi-row aggregation feature of the sub table:
+For aggregation over multiple tables, OpenMLDB extends the standard WINDOW syntax and adds [WINDOW UNION](../reference/sql/dql/WINDOW_CLAUSE.md#window-union) syntax. 
+WINDOW UNION supports splicing multiple pieces of data from the secondary table to form a window on secondary table.
+Based on the secondary table splicing window, it is convenient to construct the multi-row aggregation feature of the secondary table. 
+Similarly, two steps need to be completed to construct the multi-row aggregation feature of the secondary table:
 
-- Step 1: Define the sub table splicing window.
-- Step 2: Construct the multi-row aggregation feature of the sub table on the sub table splicing window.
+- Step 1: Define the secondary table splicing window.
+- Step 2: Construct the multi-row aggregation feature of the secondary table on the window constructed in Step 1.
 
 ## 3.1 Step 1: Define the Sub Table Splicing Window
```

Comment:
should I change this h2 format to h3? line 169 likewise?
If the change is necessary, then I will change both the zh and en version